docs/install/index.md

```diff
@@ -45,20 +38,20 @@ Add any missing requirements, including disk space.
     We strongly recommend using a recent version of your operating system released within the last 3 years.
     Older systems might not be supported.
 -   Python 3.7, 3.8, or 3.9.
--   Node.js 14 or 16.
--   For Plone images, a container engine to develop, manage, and run OCI containers.
+
+Additional requirements might be needed depending on your choice of installation method.
 
 
 (install-index-choose-installation-method-label)=
 
-## Choose an installation method
+### Choose an installation method
```

Comment:
I would put this just after "getting started" and remove the "requirements" section. Requirements depends on the installation method "docker" or "from source".

Reply:
OK, that answers my previous question. I'll fix that now.

Reply:
See 51f3f20a64781919bd3e7ba5c90530c1497bd6fc